README.md

```diff
@@ -35,31 +37,77 @@ have different degree of access throughout the application.
 
 ---
 
-### Deployment
+## 3. Deployment
+
+### Alternative 1: Download/Clone Repository
+
+- Download the [Latest Release](https://github.com/Patlenlix/CrimeDatabase/releases)
+    - Unzip the compressed file where you want to save the application
 
-1. Clone/Fork this repo in your favorite IDE
-    - Go to the folder where you want the application to save 
+**OR**
+
+- Clone the repository
+    - Go to the folder where you want the application to save
     - Run the following from your Console: `git clone https://github.com/Patlenlix/CrimeDatabase.git`
-3. Install Docker Desktop
-4. Run the application:
-    - Go to the folder of the application 
+
+**RUN APPLICATION: ALTERNATIVE 1**
+
+1. Install [Docker Desktop](https://www.docker.com/products/docker-desktop/) (and run it)
```

Comment:
It didn't work the first time I ran it, I got this error: ERROR: Get "https://registry-1.docker.io/v2/": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers). 

I remember Martin mentioning something about this problem, so I run it twice, and it worked the secound time. Maybe we should add somthing about this in the instructions?

Reply:
You mean while installing Docker desktop?

Reply:
no (sorry for bad marking of lines), runnig docker-compose up

Reply:
Ok, I see. The problem lies in us not having an image right now. Will be solved when we make a new release.